extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -209,17 +217,24 @@ export class ResourceTypePickerDialog extends DialogBase {
 				if (this.toolRefreshTimestamp !== currentRefreshTimestamp) {
 					return;
 				}
+				let enableInstallButton = false;
 				const messages: string[] = [];
 				this._toolsTable.data = toolRequirements.map(toolRef => {
 					const tool = this.toolsService.getToolByName(toolRef.name)!;
-					if (!tool.isInstalled) {
+					if (!tool.isInstalled && !tool.autoInstallSupported) {
 						messages.push(localize('deploymentDialog.ToolInformation', "{0}: {1}", tool.displayName, tool.homePage));
```

Comment:
So if all the tools support auto-install isn't this going to result in the OK button being enabled? Since we'd never push messages into the list.